components/builder-web/app/origin/origin-page/origin-settings-tab/origin-settings-tab.component.scss

```diff
@@ -0,0 +1,42 @@
+.title-description {
```

Comment:
We should also prefix this filename with an underscore: http://sass-lang.com/guide

(I suspect this might be how you got those two stowaways in `./assets`.)